types/babar/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for babar 0.2
+// Project: https://github.com/stephan83/babar#readme
+// Definitions by: Matt Bachmann <https://github.com/Bachmann1234>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+    caption?: string;
+    color?: 'yellow' | 'cyan' | 'white' | 'magenta' | 'green' | 'red' | 'grey' | 'blue' | 'ascii';
+    grid?: string;
```

Comment:
grid is a color, I don't know if the color type can be reused or if the list should be duplicated